lib/api.js

```diff
@@ -164,6 +165,11 @@ PlexAPI.prototype._request = function _request(options) {
 
         resolveValue = body;
 
+        if (response.statusCode === 403 && !options.overrideToken && self.authToken) {
```

Comment:
Have you tested these `401` / `403` statuses in depth? I'm particularly curious if `403` is actually exclusive to home user/pin auth needed or if it could also be missing username/password as the `401` code below handles..